gallery_dl/extractor/gelbooru_v02.py

```diff
@@ -57,6 +57,31 @@ def _extended_tags(self, post, page=None):
                 tags[tag_type].append(text.unquote(tag_name))
             for key, value in tags.items():
                 post["tags_" + key] = " ".join(value)
+        return page
+
+    def _notes(self, post, page=None):
+        if not page:
+            url = "{}/index.php?page=post&s=view&id={}".format(
+                self.root, post["id"])
+            page = self.request(url).text
+        notes = []
+        notes_data = text.extract(page, '<section id="notes"', '</section>')[0]
+        if not notes_data:
+            return
+
+        note_iter = text.extract_iter(notes_data, '<article', '</article>')
+        extr = text.extract
+        for note_data in note_iter:
+            note = {
+                "width": int(extr(note_data, 'data-width="', '"')[0]),
+                "height": int(extr(note_data, 'data-height="', '"')[0]),
+                "x": int(extr(note_data, 'data-x="', '"')[0]),
+                "y": int(extr(note_data, 'data-y="', '"')[0]),
+                "text": extr(note_data, 'data-body="', '"')[0],
```

Comment:
This should probably be called `body` to be consistent with Danbooru's (and other boorus'?) naming scheme for note data.

<details>
  <summary>Danbooru Note</summary>

```
        {
          "body": "Staff",
          "created_at": "2021-04-12T12:08:19.267-04:00",
          "height": 24,
          "id": 2937185,
          "is_active": true,
          "post_id": 4451394,
          "updated_at": "2021-04-12T12:09:09.685-04:00",
          "version": 1,
          "width": 40,
          "x": 1075,
          "y": 335
        }
```
</details>

Reply:
done!